Automattic/woocommerce-payments
# Adjust positioning of BNPL messaging on the classic cart page.

Fixes #8661

#### Changes proposed in this Pull Request
When the Woo Gift Card extension is active on a store, the gift card field appears above the BNPL messaging. We'd like to keep the BNPL messaging as close to the cart total as possible.

#### Testing instructions





* Download and enable Gift Card extension
* Enable desired BNPL payment methods
* Shop and add product to cart
* Observe in the classic cart that the PMME messaging is above the gift card code field

Before:
<img width="500" alt="Screenshot 2024-04-18 at 3 58 04 PM" src="https://github.com/Automattic/woocommerce-payments/assets/1558827/96296097-3413-40db-b386-ceb664115022">


After:
<img width="500" alt="Screenshot 2024-04-18 at 3 53 36 PM" src="https://github.com/Automattic/woocommerce-payments/assets/1558827/03d0c7b5-dd5c-47f2-8821-607cd8d22322">


-------------------


**Post merge**
